src/map/map.hpp

```diff
@@ -1253,7 +1253,7 @@ bool map_setmapflag_sub(int16 m, enum e_mapflag mapflag, bool status, union u_ma
 #define CHK_ELEMENT(ele) ((ele) > ELE_NONE && (ele) < ELE_MAX) /// Check valid Element
 #define CHK_ELEMENT_LEVEL(lv) ((lv) >= 1 && (lv) <= MAX_ELE_LEVEL) /// Check valid element level
 #define CHK_RACE(race) ((race) > RC_NONE_ && (race) < RC_MAX) /// Check valid Race
-#define CHK_RACE2(race2) ((race2) >= RC2_NONE && (race2) < RC2_MAX) /// Check valid Race2
+#define CHK_RACE2(race2) ((race2) > RC2_NONE && (race2) < RC2_MAX) /// Check valid Race2
```

Comment:
```suggestion
```
Please remove the function(s) completely and move the logic into PC_BONUS_CHK_RACE2